plutus-core/README.md

```diff
@@ -36,7 +36,7 @@ The Haskell package `plutus-core` implements a range of functionality for manipu
 
 ### Parser and pretty printer
 
-The lexer & parser are based on Alex & Happy and the pretty printer uses the `prettyprinter` package. Names (identifiers) are interned using uniques as per `PlutusCore.Name`. They are also parameterised with an attribute used differently in different stages.
```

Comment:
there are a bunch of unrelated changes here at the end, not the end of the world, but tidier to do those separately.

Reply:
Moved them to https://github.com/input-output-hk/plutus/pull/4553.